fgpyo/vcf/tests/test_builder.py

```diff
@@ -81,9 +81,11 @@ def _get_random_variant_inputs(
         key: (
             random_generator.randint(0, 100)
             if value_type == VcfFieldType.INTEGER
-            else random_generator.uniform(0, 1)
-            if value_type == VcfFieldType.FLOAT
-            else random_generator.choice(["Up", "Down"])
+            else (
+                random_generator.uniform(0, 1)
+                if value_type == VcfFieldType.FLOAT
+                else random_generator.choice(["Up", "Down"])
+            )
```

Comment:
I don't understand the need for this change. The original seems more pythonic to me.

Reply:
I can try to revert the change and see if black complains.

Reply:
I feel like a series of v1 if c1 else v2 if c2 else c3 is pretty typical for python. But the version black made isn't some unreadable abomination. If you don't care either way I'd say try to revert it. If black doesn't let you, that's that. I approved.